WordPressKit/WordPressOrgXMLRPCApi.swift

```diff
@@ -26,6 +28,14 @@ open class WordPressOrgXMLRPCApi: NSObject {
     ///
     @objc public static let minimumSupportedVersion = "4.0"
 
+    private lazy var urlSession: URLSession = makeSession(configuration: .default)
+    private lazy var uploadURLSession: URLSession = {
+        if self.backgroundUploads {
+            return makeSession(configuration: .background(withIdentifier: self.backgroundSessionIdentifier))
+        }
+        return urlSession
```

Comment:
Also a nitpick:

```suggestion
        guard backgroundUploads else {
            return urlSession
        }

        return makeSession(configuration: .background(withIdentifier: backgroundSessionIdentifier))
```

I can be pedantic about `guard` for early returns and I can see how the code is basically the same as before. But I find it to be a much clearer pattern.

Reply:
I have changed this implementation to use ternary conditional operation which seems cleaner. https://github.com/wordpress-mobile/WordPressKit-iOS/pull/717/commits/b6506ceea28e1895d49f92777e8e32b0406f8890

Reply:
On nice. It also uses the implicit `return`. 👨‍🍳 💋